src/profile-logic/marker-data.js

```diff
@@ -261,6 +261,19 @@ export function getTracingMarkers(
           marker.dur = time - marker.start;
         }
         tracingMarkers.push(marker);
+      } else {
+        // Tracing markers without an interval property are "event" markers.
+        // In Gecko C++ code, these are created using
+        // PROFILER_TRACING("Category", "Name", TRACING_EVENT)
+        // Add a marker with a zero duration for them.
+        const marker = {
+          start: markers.time[i],
+          dur: 0,
+          name: stringTable.getString(markers.name[i]),
+          title: null,
+          data,
+        };
+        tracingMarkers.push(marker);
```

Comment:
My small concern is that it's the exact same code as the last `else` block.

Maybe you can replace `if (data.type === 'tracing')` by `if (data.type === 'tracing' && data.interval)`, so that we could actually end up to the last `else` block to handle these markers?

Reply:
Great idea, done.